amgrss_schemas/amgrss_xpaths_description.csv

```diff
@@ -40,6 +41,9 @@ xpath, description, valueType, exampleValues, mappingRequired
 /rss/channel/item/media:content/@kind, media kind of the resource, xs:string, "preview, original, processed", no
 /rss/channel/item/media:content/@url, should specify the direct URL to the media object, xs:anyUri, "https://www.example.com/video.mp4", no
 /rss/channel/item/media:content/@width, the width of the media object, xs:integer, "1920", no
+/rss/channel/item/media:content/@targetPlatformIds, Comma separated list of AMGPLT platform id as defined in Amagi Master, xs:string, "AMGPLT001", no
```

Comment:
Since this is a comma seperated list, the example values should show that. Something like "AMGPLT001,AMGPLT005,AMGPLT020" for all three newly added fields.